### PR TITLE
Update Student Orgs

### PR DESCRIPTION
**Pull request by Sydney-tran**
### Summary 

This PR implements the new student orgs cards design created for the MVP launch. Since we have minimal information about the student orgs, our team decided that instead of having a modal pop up with information for each student org, the student orgs cards themselves should contain all the information. So this PR removes the modals and adds the tags and the website button to each student org card. Currently, there is no data for the tags in the firebase, but once the information is added, the tags should appear correctly.

### CODE REVIEW

**Review by syoung212 — COMMENTED** [review]
Great work! The student orgs page looks very clean and it looks similar to the design. I think the only concern is that for orgs with very long names (ex. American Society of Landscaper Architecture Cornell Chapter), there may be a problem when the tags pop up, especially because currently the box seems to be already very full with just the website icon in it. For this, I think you may need to adjust the font size of the title for this box. Other than that, good job!

**Review by pranavig9 — APPROVED** [review]
Looks good !

File: design-at-cornell/src/components/ButtonStyles.ts [hunk 1]
```diff
@@ -114,3 +114,28 @@ export const LargeRectangularButton = styled.div`
...
   }
 `;
+
+export const SmallRectangularButton = styled.div`
+  display: flex;
+  width: 102px;
+  height: 34px;
+  background: white;
+  border-radius: 4px;
+  box-shadow: 0px 0px 5px 2px #e6e6e6;
```
Comment (katrina433) [inline design-at-cornell/src/components/ButtonStyles.ts]: Add the color hex code to `colors.ts` and import it here

File: design-at-cornell/src/components/ButtonStyles.ts [hunk 2]
```diff
@@ -114,3 +114,28 @@ export const LargeRectangularButton = styled.div`
...
+    min-width: 16%;
+    height: auto;
+  }
+
+  p {
+    width: fit-content;
+    font-weight: 600;
+    font-size: 13px;
+    line-height: 15px;
+    color: #6b6b6b;
```
Comment (katrina433) [inline design-at-cornell/src/components/ButtonStyles.ts]: Add this hex code to `colors.ts` too

File: design-at-cornell/src/components/DashboardElementStyles.ts [hunk 3]
```diff
@@ -9,8 +9,8 @@ export const ElementContainer = styled.div`
   height: 197px;
   cursor: pointer;
   background: white;
-  border: 1px solid;
   border-radius: 8px;
+  box-shadow: 0px 0px 5px 2px #e6e6e6;
```
Comment (katrina433) [inline design-at-cornell/src/components/DashboardElementStyles.ts]: and here too

**Review by katrina433 — COMMENTED** [review]
Looks good! I left small comments to move color hex codes to the constants file, but otherwise it's good to go!

**Review by katrina433 — APPROVED** [review]
Looks nice! Great work

**Pull request merged**